src/Tickets/Seating/Service/Layouts.php

```diff
@@ -337,4 +343,76 @@ public function delete( string $layout_id, string $map_id ): bool {
 
 		return false;
 	}
+
+	/**
+	 * Updates the capacity of all posts for the given layout IDs.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<string,int> $updates The layout ID to seats count map.
+	 *
+	 * @return int|false The number of posts updated, or `false` on failure.
+	 */
+	public function update_posts_capacity( array $updates ) {
+		if ( empty( $updates ) ) {
+			return 0;
+		}
+
+		$total_updated = 0;
+
+		$ticketable_post_types = (array) tribe_get_option( 'ticket-enabled-post-types', [] );
```

Comment:
Let's use this maybe: https://github.com/the-events-calendar/event-tickets/blob/a968f0321ff27243af55dca4e04a4b065ebaa530/src/Tribe/Main.php#L1044